project_2/src/main.cpp

```diff
@@ -0,0 +1,22 @@
+#include "simulator.hpp"
+#include "express_pass_server.hpp"
+#include "express_pass_switch.hpp"
+
+int main(int argc, char* argv[]) {
+    std::uint32_t amount_of_senders = 10;
```

Comment:
English: the word `amount` refers to uncountable nouns only, e.g. "amount of water". For countable nouns the word `number` should be used.